frontend/src/container/LogsExplorerViews/index.tsx

```diff
@@ -116,6 +120,10 @@ function LogsExplorerViews({
 	const [logs, setLogs] = useState<ILog[]>([]);
 	const [requestData, setRequestData] = useState<Query | null>(null);
 	const [showFormatMenuItems, setShowFormatMenuItems] = useState(false);
+	const [queryId, setQueryId] = useState<string>(
+		Math.random().toString(36).slice(2),
```

Comment:
Can we not use a fixed constant here ?

Reply:
no we cannot use a fixed constant as multiple users opening the UI at the same time will have the same id and the socket will get attached to wrong queries if we have the same initial ID for everyone